topics/galaxy-interface/tutorials/workflow-fairification/tutorial.md

```diff
@@ -0,0 +1,668 @@
+>
+> Which metadata is supported in Galaxy workflow interface?
+>
+> > <solution-title></solution-title>
+> > Galaxy workflow interface supports:
+> > - Name
+> > - Version
+> > - Annotation: These notes will be visible when this workflow is viewed.
+> > - License
+> > - Creator: Either a person or an organization. 
+> > - Tags: Apply tags to make it easy to search for and find items with the same tag. 
+> {: .solution}
+{: .question}
+
+Galaxy workflow interface supports some metadata. Are they enough to fulfill Tip 2 and also Tip 1 *"Register the workflow."*. WorkflowHub, a workflow registry we will explain more later, supports the [following metadata](https://about.workflowhub.eu/docs/metadata-list/):
+
+Name | Description | Mandatory
+--- | --- |
+**Title** | This field is mandatory and is with some workflow types pre-filled with the title of the workflow. | Yes
+**Description** | If a CWL (abstract) file is given, the description will be parsed automatically out of the doc attribute. In any other case this field can be used to write some documentation that will be shown on the workflow page. | No
+**Source** | If the workflow came from an external repository (i.e. GitHub), you can include its original URL here. | No
+**Maturity** | This field can be used to specify in which maturity state the workflow is. The two available options are: `work-in-progress`, `stable` | No
+**Teams** | Every workflow registration is linked to one or more teams. | Yes
+**Licence** | The standard licence is Apache Software Licence 2.0. If you did not make the workflow yourself, be sure that the licence corresponds to the licence where you took the workflow from (for example github licences). | No
+**Sharing** | Specify who can view the summary, get access to the content, and edit the Workflow. This is possibly already filled in according to the selected project. | No
+**Tags** | Choose an appropriate tag for your workflow. Please check if your tag is already available and use the existing one if so. If you make a new tag, keep it simple without capitals or spaces. For example all new covid-19 workflows need to be tagged with covid-19. | No
+**Creators** | This is an important section where all the people that were involved in making / publishing this workflow are listed. | No
+
+> <question-title></question-title>
+>
+> 1. Which WorkflowHub metadata is supported in Galaxy workflow interface?
+> 2. Which values should we put for our workflow?
+>
+> > <solution-title></solution-title>
+> >
+> > Workflow metadata | Galaxy metadata | Mapping workflow value
+> > --- | --- | ---
+> > Title | Name | Quality control and mapping
+> > Description | Annotation | Workflow runs quality control and mapping of paired-end short-reads data.
+> > Source | *Not supported in Galaxy workflow interface* | 
+> > Teams | *Not supported in Galaxy workflow interface* | 
+> > License | License | MIT License
+> > Sharing | *Not supported in Galaxy workflow interface* | 
```

Comment:
We can share workflows, it just works differently between both systems.

I'm not sure if its super useful to compare both metadata systems. Things like Teams/Source etc. only makes sense on the workflowhub side.